root/static/scripts/relationship-editor/common/dialog.js

```diff
@@ -313,7 +314,7 @@ const RE = MB.relationshipEditor = MB.relationshipEditor || {};
 
         keydownEvent(data, event) {
             if (event.isDefaultPrevented()) {
-                return;
+                return false;
```

Comment:
This might be an exception to my event handler comments above, since the knockout-delegatedEvents plugin actually does use the `true` value: https://github.com/rniemeyer/knockout-delegatedEvents/blob/309b1877b3bd33bbfb20a9cf197e94e7e777a644/src/knockout-delegatedEvents.js#L158

So returning false is probably fine here, since this plugin is confusing anyway. :) (And we'll be changing all this code when it's converted to React.)